src/eval/Runner.ml

```diff
@@ -101,10 +99,11 @@ let input_state_json filename =
 
 (* Add balance to output json and print it out *)
 let output_state_json balance field_vals =
-  let bal_lit = (balance_label, UintLit (Uint128L balance)) in
+  let bal_lit = (balance_label, JSON.JSONLiteral.UintLit (Uint128L balance)) in
```

Comment:
Would be easy to turn `JSON.JSONLiteral.UintLit` into `JSON.Literal.UintLit`?

Reply:
Maybe not, but if it is a problem then I think we can just use `JSON.Literal` here. I got lost a bit lost in the JSON files, but once we have multiple instantiations of `Name` then it should be obvious what needs to happen.

Reply:
Sure, let us postpone this then.